_maps/map_files/cyberiad/z3.dmm

```diff
@@ -1,357 +1,234 @@
+"bR" = (/obj/structure/cable{d1 = 1; d2 = 8; icon_state = "1-8"},/obj/machinery/light,/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"bS" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/obj/machinery/atmospherics/pipe/simple/visible,/obj/effect/decal/warning_stripes{icon_state = "S-corner"},/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "dark"; dir = 5},/area/turret_protected/tcomsat)
+"bT" = (/obj/structure/cable{d1 = 1; d2 = 4; icon_state = "1-4"; tag = ""},/obj/machinery/light,/obj/machinery/light{dir = 8},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"bU" = (/obj/machinery/door/airlock/external{frequency = 1382; icon_state = "door_locked"; id_tag = "telecomms_server_room_inner"; locked = 1; name = "Telecomms Server Room"; req_access = null; req_access_txt = "13"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/obj/machinery/atmospherics/pipe/simple/visible,/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "dark"; dir = 5},/area/turret_protected/tcomfoyer)
+"bV" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/light{dir = 8},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
 "bW" = (/obj/structure/lattice,/obj/structure/grille,/turf/space,/area/space)
-"bX" = (/obj/machinery/power/solar,/obj/structure/cable{icon_state = "0-4"; d2 = 4},/turf/simulated/floor/plasteel/airless{icon_state = "solarpanel"},/area/space)
-"bY" = (/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/turf/simulated/floor/plating/airless,/area/space)
-"bZ" = (/obj/machinery/power/solar,/obj/structure/cable{d2 = 8; icon_state = "0-8"},/turf/simulated/floor/plasteel/airless{icon_state = "solarpanel"},/area/space)
-"ca" = (/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/turf/simulated/floor/plating/airless,/area/space)
+"bX" = (/obj/structure/lattice,/obj/structure/grille,/obj/structure/lattice,/turf/space,/area/space)
+"bY" = (/obj/machinery/camera{c_tag = "Telecomms Control Room"; dir = 2; network = list("Telecomms")},/obj/structure/table,/obj/item/folder/yellow,/obj/item/folder/yellow,/obj/item/radio/intercom/pirate{desc = "A standard station intercom, with all channels unlocked. Useful for testing scripts."; name = "unlocked intercom"; pixel_x = 0; pixel_y = 20},/turf/simulated/floor/plasteel{dir = 1; icon_state = "darkred"},/area/tcommsat/computer)
+"bZ" = (/obj/machinery/computer/telecomms/monitor,/obj/effect/decal/warning_stripes/white,/turf/simulated/floor/plasteel{dir = 9; icon_state = "darkred"},/area/tcommsat/computer)
```

Comment:
This room is far too small, on numerous occasions I had 3 people in there at the same time. 1.5x the size would be good.

Reply:
Which one

Reply:
Me: "do you want me to review by line?"
You: "yes"
Argh! `/area/tcommsat/computer` -> The one with the Traffic Control Console.

Reply:
Fixed